ere-guests/rlp-encoding-length/sp1/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "succinct-rlp-encoding-length"
+version = "0.1.0"
+edition = "2024"
+rust-version = "1.85"
+license = "MIT OR Apache-2.0"
+
+[dependencies]
+sp1-zkvm = "5.0.5"
+
+tracing = "*"
+tracing-subscriber = "*"
+serde = { version = "1.0", default-features = false, features = ["derive"] }
+serde_with = "3"
+
+reth-ethereum-primitives = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf", features = [
+    "serde",
+    "serde-bincode-compat",
+] }
+alloy-consensus = { version = "1.0.18", default-features = false }
+reth-primitives-traits = { git = "https://github.com/kevaundray/reth", rev = "75092d6aa098311db96c06b97a252a6f2b42dbaf" }
```

Comment:
This is temporary until we support thew new `ere`-Dockerization, since now guests must be self contained.

After we do the change, we'll replace all these dependencies with `guest-libs.workspace = true`.